app/views/documents/_filter_form.html.erb

```diff
@@ -1,21 +1,19 @@
 <% unless defined?(filters); filters = []; end %>
 <% document_type_plural = document_type.to_s.pluralize %>
 <div class="filter-form-block">
-  <div class="inner-block">
-    <h2><%= t('document_filters.description').html_safe %></h2>
-    <form method="get" action="<%= send(:"#{document_type_plural}_path") %>" id="document-filter" class="js-document-filter filter-form">
-      <fieldset>
-        <legend class="visuallyhidden"><%= t("support.filter") %> <%= document_type_plural %></legend>
+  <h2><%= t('document_filters.description').html_safe %></h2>
```

Comment:
Can [heading](https://components.publishing.service.gov.uk/component-guide/heading) could be used here?